reference/5.1/Microsoft.PowerShell.Security/ConvertTo-SecureString.md

```diff
@@ -37,90 +34,106 @@ ConvertTo-SecureString [-String] <String> [-Key <Byte[]>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **ConvertTo-SecureString** cmdlet converts encrypted standard strings into secure strings.
-It can also convert plain text to secure strings.
-It is used with ConvertFrom-SecureString and Read-Host.
-The secure string created by the cmdlet can be used with cmdlets or functions that require a parameter of type SecureString.
-The secure string can be converted back to an encrypted, standard string using the ConvertFrom-SecureString cmdlet.
-This enables it to be stored in a file for later use.
 
-If the standard string being converted was encrypted with **ConvertFrom-SecureString** using a specified key, that same key must be provided as the value of the *Key* or *SecureKey* parameter of the **ConvertTo-SecureString** cmdlet.
+The `ConvertTo-SecureString` cmdlet converts encrypted standard strings into secure strings. It can
+also convert plain text to secure strings. It is used with `ConvertFrom-SecureString` and
+`Read-Host`. The secure string created by the cmdlet can be used with cmdlets or functions that
+require a parameter of type SecureString. The secure string can be converted back to an encrypted,
+standard string using the `ConvertFrom-SecureString` cmdlet. This enables it to be stored in a file
+for later use.
+
+If the standard string being converted was encrypted with `ConvertFrom-SecureString` using a
+specified key, that same key must be provided as the value of the **Key** or **SecureKey** parameter
+of the `ConvertTo-SecureString` cmdlet.
 
 ## EXAMPLES
 
 ### Example 1: Convert a secure string to an encrypted string
 
-```
+This example shows how to create a secure string from user input, convert the secure string to an
+encrypted standard string, and then convert the encrypted standard string back to a secure string.
+
+```powershell
```

Comment:
While updating this, clean up the prompts?

Reply:
stet